core/cap-0036.md

```diff
@@ -0,0 +1,189 @@
+  on the `sourceAccount`.
+
+## Design Rationale
+The rationale for this proposal extends the rational as described in CAP-35. Claimable balances store value on the network and the same events of regulatory action, erroneous transaction, or loss of custody of a claimaint account, can result in an issuer needing to reissue an asset stored within a claimable balance. CAP-35 adds this capability to assets stored in accounts, and this proposal provides the same functionality to claimable balances.
+
+CAP-35 allows an issuer to clawback specific amounts within an account but this proposal limits clawback to the full amount within the claimable balance. Claimable balances are immutable and there would be massive downstream impacts to changing the amount within a claimable balance.
+
+Claimable balances cannot be recreated after they are destroyed with clawback because their claimable balance ID is generated from the transaction that creates them, and to recreate the same claimable balance would duplicate events in downstream systems. Issuers can reissue the asset in a new claimable balance configured with the same claimaints and predicates to recreate the claimable balance, but the claimable balance ID will not be identical. 
+ 
+### Reusing the AUTH_REVOCABLE flag
+The account `AUTH_REVOCABLE` flag allows the issuer to indicate that it has
+control over the use of the asset on the network. By including the
+`AUTH_REVOCABLE` flag in account flags, account owners may review the
+revocability of an asset issued by the issuer and have the choice to avoid this
+type of asset if they object to the implied trust in the issuer. Clawback is
+another form of an issuer revoking use of an asset with fine control over the
+exact amount that the issuer is taking out of active circulation.
+
+### Threshold
+The clawback operation requires a medium threshold signature because it is
+changing the balance of an account and is more aligned with impact to a payment
+operation than an allow trust operation.
+
+## Protocol Upgrade Transition
+
+### Backwards Incompatibilities
+
+The change does not have an affect on previous assets, accounts, or transaction 
+structure. It should not cause a breaking change in existing implementations. 
+
+The change does make it possible for claimable balances to be destroyed, and
+claimable balances could form an important part of series of transactions that
+form a logical contract. The use of a clawback could break existing contracts
+that did not consider the possibility of clawback. Issuers today who use the
+`AUTH_REVOCABLE` flag get the same capability by freezing claimaint accounts.
+Auth revocation is reversable meaning an accidental auth revocation if reversed
+would have no impact on a planned series of transactions that form a logical
+contract. However, a clawed back claimable balance recreated would have a new
+claimable balance ID and would not be claimable with pre-signed or
+pre-authorized transactions.
```

Comment:
@MonsieurNicolas This backwards incompatibility could be avoided if claimable balances were mutable and instead of destroying the claimable balance entry its amount was altered. I realize from the mailing list discussion thread you mentioned that mutability of this ledger entry is undesirable. This is the tradeoff.

Reply:
Rather than this be a PR comment, I've placed this comment into the proposal in the design rationale section so it is visible when this is discussed on the mailing list. 94998f1cb8821fdbff1e54202d6636e6b89897fd